include/prereq.mk

```diff
@@ -116,3 +116,27 @@ define SetupHostCommand
 
   $$(eval $$(call Require,$(1),$(if $(2),$(2),Missing $(1) command)))
 endef
+
+define SetupHostPython
```

Comment:
A dupe of SetupHostCommand; I had initially tried to patch the original symlink section to do
the python stuff differently if `$(1)` was `python*` but that didn't quite work; maybe not the right
amount of $ or something.

If this approach is approved for inclusion in master I'm more than willing to try a 'better' way of
doing this handling.

Reply:
try case statement

```
case $(1) in
   "python"*) venv ....
   *) ln -sf ....
esac
```

Reply:
applying:
```patch
diff --git a/include/prereq.mk b/include/prereq.mk
index f2fc4d944d..ef309ff71e 100644
--- a/include/prereq.mk
+++ b/include/prereq.mk
@@ -106,7 +106,10 @@ define SetupHostCommand
                                command -v "$$$$$$$${cmd%% *}")"; \
                        if [ -x "$$$$$$$$bin" ] && eval "$$$$$$$$cmd" >/dev/null 2>/dev/null; then \
                                mkdir -p "$(STAGING_DIR_HOST)/bin"; \
-                               ln -sf "$$$$$$$$bin" "$(STAGING_DIR_HOST)/bin/$(strip $(1))"; \
+                               case $(1) in \
+                                       "python"*) "$$$$$$$$bin" -m venv "$(STAGING_DIR_HOST)/" ;; \
+                                       *) ln -sf "$$$$$$$$bin" "$(STAGING_DIR_HOST)/bin/$(strip $(1))" ;; \
+                               esac; \
                                exit 0; \
                        fi; \
                fi; \
```
seems to work, in that it creates the venv as expected and passes checks, however it throws a lot of
`grep: warning: stray \ before /` warnings.

Reply:
Hi, do you have any dm messanger ?
Like discord
I want to disguise about panfrost

Reply:
> applying:
> 
> ```diff
> diff --git a/include/prereq.mk b/include/prereq.mk
> index f2fc4d944d..ef309ff71e 100644
> --- a/include/prereq.mk
> +++ b/include/prereq.mk
> @@ -106,7 +106,10 @@ define SetupHostCommand
>                                 command -v "$$$$$$$${cmd%% *}")"; \
>                         if [ -x "$$$$$$$$bin" ] && eval "$$$$$$$$cmd" >/dev/null 2>/dev/null; then \
>                                 mkdir -p "$(STAGING_DIR_HOST)/bin"; \
> -                               ln -sf "$$$$$$$$bin" "$(STAGING_DIR_HOST)/bin/$(strip $(1))"; \
> +                               case $(1) in \
> +                                       "python"*) "$$$$$$$$bin" -m venv "$(STAGING_DIR_HOST)/" ;; \
> +                                       *) ln -sf "$$$$$$$$bin" "$(STAGING_DIR_HOST)/bin/$(strip $(1))" ;; \
> +                               esac; \
>                                 exit 0; \
>                         fi; \
>                 fi; \
> ```
> 
> seems to work, in that it creates the venv as expected and passes checks, however it throws a lot of `grep: warning: stray \ before /` warnings.

Hi, do you have any dm messanger ?
Like discord
I want to discuss about panfrost

Reply:
I'm on the libera and oftc irc networks, same name.

Reply:
Sorry, idk how to use this, and on libera when i close browser all chat delete.
Do you have any other variants of messanging?

Reply:
not that I want to post on an open forum. send me an email with your contact info (myusernamehere @ protonmail dot com)